app/projectwizard.cpp

```diff
@@ -0,0 +1,219 @@
+#include "projectwizard.h"
+#include "inpututils.h"
+
+#include "qgsproject.h"
+#include "qgsrasterlayer.h"
+#include "qgsvectorlayer.h"
+#include "qgsvectorfilewriter.h"
+#include "qgsdatetimefieldformatter.h"
+
+ProjectWizard::ProjectWizard( const QString &dataDir, FieldsModel *fieldsModel, QObject *parent )
+  : QObject( parent )
+  , mDataDir( dataDir )
+  , mFieldsModel( fieldsModel )
+{
+
+  mSettings = new QgsMapSettings();
```

Comment:
It seems this is not gettting deleted anywhere....

Best to use std::unique_ptr rather than raw pointers...